### PR TITLE
Dev 01

### PR DESCRIPTION
**Pull request by developer351**

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (neok) [inline .idea/workspace.xml:1]: move your idea to gitignore

File: app/bootstrap.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: neos
+ * Date: 29.01.16
+ * Time: 2:36
```
Comment (neok) [inline app/bootstrap.php]: delete such comments

File: index.php [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: neos
```
Comment (neok) [inline index.php]: delete such comments

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+/**
+ * Created by neos on 29.01.16.
```
Comment (neok) [inline js/script.js]: delete such comments

File: vendor/composer/ClassLoader.php [hunk 5]
```diff
@@ -0,0 +1,413 @@
+<?php
```
Comment (neok) [inline vendor/composer/ClassLoader.php:1]: vendor folter add to .gitignore

File: .idea/workspace.xml (hunk 1 above)
Comment (developer351) [inline .idea/workspace.xml:1]: ok

File: app/bootstrap.php (hunk 2 above)
Comment (developer351) [inline app/bootstrap.php]: deleted

File: index.php (hunk 3 above)
Comment (developer351) [inline index.php]: deleted

File: js/script.js (hunk 4 above)
Comment (developer351) [inline js/script.js]: deleted

File: vendor/composer/ClassLoader.php (hunk 5 above)
Comment (developer351) [inline vendor/composer/ClassLoader.php:1]: added

**Pull request merged**